moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -291,7 +291,6 @@ const ProjectComponentsMapEdit = ({
     const projectFeaturesToCreateOrUpdate = editFeatureCollection.features.map(
       feature => {
         let projectFeature = {};
-        projectFeature.status_id = 1;
```

Comment:
We no longer need to set status_id to 1 (not soft deleted) here since the default value of `is_deleted` is false on record creation.